src/browser/api/browser_view.ts

```diff
@@ -19,6 +22,12 @@ export interface BrowserViewOpts extends BrowserViewCreationOptions {
 }
 
 export async function create(options: BrowserViewOpts) {
+    // checking if the name-uuid combination is already in use
+    const { uuid, name } = options;
+    if (getWindowByUuidName(uuid, name) || getBrowserViewByIdentity({ uuid, name }) || getInfoByUuidFrame({ uuid, name })) {
+        throw new Error('Provided name-uuid combination is already in use');
```

Comment:
Can we make this message more inline with the window one? Let's also add what the uuid/name combination is in the message.

Reply:
:+1: